ftl/common/builder_runner.py

```diff
@@ -55,25 +57,23 @@ def __init__(self, args, builder, cache_version_str):
             mount=[self.base_name])
         self.builder = builder.From(self.ctx)
 
-    def GetCacheKey(self, descriptor_files):
-        descriptor = None
-        for f in descriptor_files:
-            if self.ctx.Contains(f):
-                descriptor = f
-                descriptor_contents = self.ctx.GetFile(descriptor)
-                break
-        if not descriptor:
-            logging.info('No package descriptor found. No packages installed.')
+    def GetCacheKey(self, key):
+        if key is None:
```

Comment:
Why would this be none? Could we check before this gets called insted?

Reply:
None is used a sentinel for use cached base-image, currently this is the solution I made to support this